From 2d4d14022da523110f5171fff7705cfcbf6ec63e Mon Sep 17 00:00:00 2001
From: aroflcoppter <contact@dankium.ca>
Date: Sat, 14 Jun 2025 10:22:43 -0230
Subject: [PATCH] adblock rearrange and new filters from dankium server

---
 .../core/subscription/subscription_config.cc  | 89 ++++++++++---------
 1 file changed, 48 insertions(+), 41 deletions(-)

diff --git a/components/adblock/core/subscription/subscription_config.cc b/components/adblock/core/subscription/subscription_config.cc
index ecd347781186c..0be62092be67d 100644
--- a/components/adblock/core/subscription/subscription_config.cc
+++ b/components/adblock/core/subscription/subscription_config.cc
@@ -72,16 +72,6 @@ KnownSubscriptionInfo::KnownSubscriptionInfo(
       privileged_status(privileged_status_param) {}
 
 const std::vector<KnownSubscriptionInfo>& config::GetKnownSubscriptions() {
-  // The current list of subscriptions can be downloaded from:
-  // https://gitlab.com/eyeo/filterlists/subscriptionlist/-/archive/master/subscriptionlist-master.tar.gz
-  // The archive contains files with subscription definitions. The ones we're
-  // interested in are ones that declare a [recommendation] keyword in either
-  // the "list" or "variant" key.
-  // Here's a script that parses the archive into a subscriptions.json:
-  // https://gitlab.com/eyeo/adblockplus/abc/adblockpluscore/-/blob/next/build/updateSubscriptions.js
-  // The list isn't updated very often. If it starts to become a burden to
-  // align the C++ representation, better to update it manually because it also
-  // contains visibility and first run behavior options.
   static std::vector<KnownSubscriptionInfo> recommendations = {
       {GURL(GetHost() + "easylist.txt"),
        "EasyList",
@@ -89,6 +79,54 @@ const std::vector<KnownSubscriptionInfo>& config::GetKnownSubscriptions() {
        SubscriptionUiVisibility::Visible,
        SubscriptionFirstRunBehavior::Subscribe,
        SubscriptionPrivilegedFilterStatus::Forbidden},
+      {GURL(GetHost() + "abp-filters-anti-cv.txt"),
+       "ABP anti-circumvention filters",
+       {},
+       SubscriptionUiVisibility::Visible,
+       SubscriptionFirstRunBehavior::Subscribe,
+       SubscriptionPrivilegedFilterStatus::Allowed},
+      {GURL(GetHost() + "i_dont_care_about_cookies.txt"),
+       "I don't care about cookies",
+       {},
+       SubscriptionUiVisibility::Visible,
+       SubscriptionFirstRunBehavior::Ignore,
+       SubscriptionPrivilegedFilterStatus::Forbidden},
+      {GURL(GetHost() + "fanboy-notifications.txt"),
+       "Fanboy's Notifications Blocking List",
+       {},
+       SubscriptionUiVisibility::Visible,
+       SubscriptionFirstRunBehavior::Ignore,
+       SubscriptionPrivilegedFilterStatus::Forbidden},
+      {GURL(GetHost() + "easyprivacy.txt"),
+       "EasyPrivacy",
+       {},
+       SubscriptionUiVisibility::Visible,
+       SubscriptionFirstRunBehavior::Subscribe,
+       SubscriptionPrivilegedFilterStatus::Forbidden},
+      {GURL(GetHost() + "antiadblockfilters.txt"),
+       "Anti-adblock filters",
+       {},
+       SubscriptionUiVisibility::Visible,
+       SubscriptionFirstRunBehavior::Subscribe,
+       SubscriptionPrivilegedFilterStatus::Forbidden},
+      {GURL(GetHost() + "fanboy-social.txt"),
+       "Fanboy's Social Blocking List",
+       {},
+       SubscriptionUiVisibility::Visible,
+       SubscriptionFirstRunBehavior::Ignore,
+       SubscriptionPrivilegedFilterStatus::Forbidden},
+      {GURL(GetHost() + "urlhaus-filter-agh.txt"),
+       "Anti-Malware",
+       {},
+       SubscriptionUiVisibility::Visible,
+       SubscriptionFirstRunBehavior::Subscribe,
+       SubscriptionPrivilegedFilterStatus::Forbidden},
+      {GURL(GetHost() + "yoyolist.txt"),
+       "Peter Lowe",
+       {},
+       SubscriptionUiVisibility::Visible,
+       SubscriptionFirstRunBehavior::Subscribe,
+       SubscriptionPrivilegedFilterStatus::Forbidden},
       {GURL(GetHost() + "abpindo.txt"),
        "ABPindo",
        {"id", "ms"},
@@ -240,37 +278,6 @@ const std::vector<KnownSubscriptionInfo>& config::GetKnownSubscriptions() {
        SubscriptionUiVisibility::Visible,
        SubscriptionFirstRunBehavior::SubscribeIfLocaleMatch,
        SubscriptionPrivilegedFilterStatus::Forbidden},
-      {GURL(GetHost() + "abp-filters-anti-cv.txt"),
-       "ABP filters",
-       {},
-       SubscriptionUiVisibility::Visible,
-       SubscriptionFirstRunBehavior::Subscribe,
-       SubscriptionPrivilegedFilterStatus::Allowed},
-      {GURL(GetHost() + "i_dont_care_about_cookies.txt"),
-       "I don't care about cookies",
-       {},
-       SubscriptionUiVisibility::Visible,
-       SubscriptionFirstRunBehavior::Ignore,
-       SubscriptionPrivilegedFilterStatus::Forbidden},
-      {GURL(GetHost() + ""
-                        "fanboy-notifications.txt"),
-       "Fanboy's Notifications Blocking List",
-       {},
-       SubscriptionUiVisibility::Visible,
-       SubscriptionFirstRunBehavior::Ignore,
-       SubscriptionPrivilegedFilterStatus::Forbidden},
-      {GURL(GetHost() + "easyprivacy.txt"),
-       "EasyPrivacy",
-       {},
-       SubscriptionUiVisibility::Visible,
-       SubscriptionFirstRunBehavior::Subscribe,
-       SubscriptionPrivilegedFilterStatus::Forbidden},
-      {GURL(GetHost() + "fanboy-social.txt"),
-       "Fanboy's Social Blocking List",
-       {},
-       SubscriptionUiVisibility::Visible,
-       SubscriptionFirstRunBehavior::Ignore,
-       SubscriptionPrivilegedFilterStatus::Forbidden},
       {CustomFiltersUrl(),
        "User filters",
        {},
-- 
2.43.0

