From a2725b301c7477df25329ea501d95daaf18dc12a Mon Sep 17 00:00:00 2001
From: aroflcoppter <contact@dankium.ca>
Date: Wed, 4 Jun 2025 05:42:42 -0230
Subject: [PATCH] Always enable command line on Android

---
 chrome/common/chrome_features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index b8b3aa0bd1c31..353e51c23c0eb 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -184,7 +184,7 @@ BASE_FEATURE(kCryptohomeUserDataAuthKillswitch,
 #if BUILDFLAG(IS_ANDROID)
 BASE_FEATURE(kCommandLineOnNonRooted,
              "CommandLineOnNonRooted",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 #endif
 
 #if BUILDFLAG(IS_CHROMEOS)
-- 
2.43.0

