From c6d275bec7625eb9252f8b74e734500afcca5c4a Mon Sep 17 00:00:00 2001
From: aroflcoppter <contact@dankium.ca>
Date: Wed, 4 Jun 2025 13:43:48 -0230
Subject: [PATCH 2/6] Add dank and nondank states

---
 base/feature_list.h | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/base/feature_list.h b/base/feature_list.h
index 731ea5dc22b4d..730ac80747bed 100644
--- a/base/feature_list.h
+++ b/base/feature_list.h
@@ -38,8 +38,10 @@ class FeatureVisitor;
 // NOTE: The actual runtime state may be different, due to a field trial or a
 // command line switch.
 enum FeatureState {
-  FEATURE_DISABLED_BY_DEFAULT,
-  FEATURE_ENABLED_BY_DEFAULT,
+  FEATURE_DISABLED_BY_DEFAULT = 0,
+  FEATURE_ENABLED_BY_DEFAULT = 1,
+  NONDANK_FEATURE_DISABLED = 0,
+  DANK_FEATURE_ENABLED = 1,
 };
 
 // Recommended macros for declaring and defining features and parameters:
-- 
2.43.0

