From 6732856c5fc120741e0371408daa747ff77e975f Mon Sep 17 00:00:00 2001
From: aroflcoppter <contact@dankium.ca>
Date: Wed, 4 Jun 2025 03:34:36 -0230
Subject: [PATCH 1/2] Branding for desktop

---
 chrome/app/theme/chromium/BRANDING | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/chrome/app/theme/chromium/BRANDING b/chrome/app/theme/chromium/BRANDING
index f8363d5b294fe..20ab62a0236fd 100644
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@
 COMPANY_FULLNAME=The Chromium Authors
 COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
-COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Chromium Authors. All rights reserved.
-MAC_BUNDLE_ID=org.chromium.Chromium
+PRODUCT_FULLNAME=Dankium
+PRODUCT_SHORTNAME=Dankium
+PRODUCT_INSTALLER_FULLNAME=Dankium Installer
+PRODUCT_INSTALLER_SHORTNAME=Dankium Installer
+COPYRIGHT=Copyright @LASTCHANGE_YEAR@ aroflcopper.
+MAC_BUNDLE_ID=app.dankium.browser
 MAC_CREATOR_CODE=Cr24
 MAC_TEAM_ID=
-- 
2.43.0

