From ec00745f8d9755d8e62db2df7d11055e24313494 Mon Sep 17 00:00:00 2001
From: aroflcoppter <contact@dankium.ca>
Date: Sat, 7 Jun 2025 04:49:48 -0230
Subject: [PATCH] Enable partitioning

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index d7d7795cff28e..74398d6b66f85 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -144,7 +144,7 @@ BASE_FEATURE(kSplitCodeCacheByNetworkIsolationKey,
 
 BASE_FEATURE(kPartitionConnectionsByNetworkIsolationKey,
              "PartitionConnectionsByNetworkIsolationKey",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             kEnabledByDefault);
 
 BASE_FEATURE(kSearchEnginePreconnectInterval,
              "SearchEnginePreconnectInterval",
-- 
2.43.0

