From e4cbc6e6b8e84ce39fe8602235961005215bee67 Mon Sep 17 00:00:00 2001
From: aroflcoppter <contact@dankium.ca>
Date: Wed, 4 Jun 2025 16:51:10 -0230
Subject: [PATCH] Enable strict site isolation

---
 chrome/common/chrome_features.cc | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index c948e61ad53ed..bd82588885353 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -1391,13 +1391,11 @@ const base::FeatureParam<base::TimeDelta> kSCTLogMaxIngestionRandomDelay{
 //
 // TODO(alexmos): Move this and the other site isolation features below to
 // browser_features, as they are only used on the browser side.
+const base::FeatureState kEnabledByDefault = base::FEATURE_ENABLED_BY_DEFAULT;
+
 BASE_FEATURE(kSitePerProcess,
              "SitePerProcess",
-#if BUILDFLAG(IS_ANDROID) && !BUILDFLAG(ENABLE_ANDROID_SITE_ISOLATION)
-             base::FEATURE_DISABLED_BY_DEFAULT
-#else
-             base::FEATURE_ENABLED_BY_DEFAULT
-#endif
+             kEnabledByDefault
 );
 
 // The default behavior to opt devtools users out of
-- 
2.43.0

