From 2f7a8d26809b990a222528583b63a8302dd7fa92 Mon Sep 17 00:00:00 2001
From: aroflcoppter <contact@dankium.ca>
Date: Wed, 4 Jun 2025 16:42:20 -0230
Subject: [PATCH 31/39] Enable partitoning

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index ad9d8a2a9081a..f65fbe925feed 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -138,7 +138,7 @@ BASE_FEATURE(kSplitCodeCacheByNetworkIsolationKey,
 
 BASE_FEATURE(kPartitionConnectionsByNetworkIsolationKey,
              "PartitionConnectionsByNetworkIsolationKey",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::DANK_FEATURE_ENABLED);
 
 BASE_FEATURE(kPostQuantumKyber,
              "PostQuantumKyber",
-- 
2.43.0

