From c51fd32ed463742cf94feb8f075df71268abeea6 Mon Sep 17 00:00:00 2001
From: aroflcoppter <contact@dankium.ca>
Date: Sat, 14 Jun 2025 11:51:51 -0230
Subject: [PATCH] Change default profile name on desktop

---
 chrome/app/profiles_strings.grdp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/chrome/app/profiles_strings.grdp b/chrome/app/profiles_strings.grdp
index f9776dace4b9f..45022e76cddc3 100644
--- a/chrome/app/profiles_strings.grdp
+++ b/chrome/app/profiles_strings.grdp
@@ -45,10 +45,10 @@
     Verify it's you
   </message>
   <message name="IDS_AVATAR_BUTTON_WORK" desc="Short label for the avatar button when the profile is a managed profile in an enterprise environment, such as a business, government organization, or other workplace." meaning="Using a managed profile, as for a workplace">
-    Work
+    Dankium
   </message>
   <message name="IDS_AVATAR_BUTTON_SCHOOL" desc="Short label for the avatar button when the profile is a school profile." meaning="Using a school profile">
-    School
+    Dankium
   </message>
   <message name="IDS_AVATAR_BUTTON_GREETING" desc="Greeting that appears at the top of Chrome after the user signs in. It becomes part of the entrypoint for the Chrome Profile Menu. The greeting is casual and friendly, and it invokes the user's name.">
     Hi, <ph name="PROFILE_NAME">$1<ex>User</ex></ph>
@@ -125,7 +125,7 @@
     Sign in to get your passwords and more on all your devices
   </message>
   <message name="IDS_PROFILE_MENU_PROFILE_IDENTIFIER_WITH_SEPARATOR" desc="The user name and profile name are separated by a symbol, in this case bullet •, with a white space before it and a white space after it. If the bullet • resembles another symbol in the language, please translate the bullet • as a symbol (A) that is not easily mistaken for a number or letter in the language and (B) that is typically used to separate elements.">
-    <ph name="USER_NAME">$1<ex>John</ex></ph> • <ph name="PROFILE_NAME">$2<ex>Work</ex></ph>
+    <ph name="USER_NAME">$1<ex>John</ex></ph> • <ph name="PROFILE_NAME">$2<ex>Dankium</ex></ph>
   </message>
   <message name="IDS_PROFILE_MENU_PROFILE_MANAGED_HEADER" desc="Disclaimer indicating that the profile is managed">
     This profile is managed
-- 
2.49.0.windows.1

